PsiphonUITests/SnapshotHelper.swift

```diff
@@ -149,10 +155,17 @@ open class Snapshot: NSObject {
 
         print("snapshot: \(name)") // more information about this, check out https://docs.fastlane.tools/actions/snapshot/#how-does-it-work
 
-        sleep(1) // Waiting for the animation to be finished (kind of)
+        if Snapshot.waitForAnimations {
+            sleep(1) // Waiting for the animation to be finished (kind of)
```

Comment:
We could alternatively use `[UIView setAnimationsEnabled:NO];` in the app code or set `XCUIApplication` instance `.launchEnvironment = ["animations": "0"]` before calling `app.launch()` here: https://github.com/Psiphon-Inc/psiphon-ios-vpn/blob/5b5884ccbee19b4bb46ddf3fe1236c3227df77b2/PsiphonUITests/PsiphonUITests.swift#L35

Reply:
Currently animations are not a problem for us while generating screenshots, but I added the functionality for disabling animations, and enabled it, in https://github.com/Psiphon-Inc/psiphon-ios-vpn/pull/255/commits/5cb09920a303290d26bfe327ec77bf2f8fecd6d3 because this functionality seems desirable if we end up having animations on a screen we want to capture.